source/includes/pipeline_config/_004-git-material.md.erb

```diff
@@ -0,0 +1,36 @@
+<%
+json ='{
+  "materials": [
+    {
+      "type": "git",
+      "attributes": {
+        "url": "git@github.com:example/sample_repo.git",
+        "destination": "code",
+        "filter": {
+          "ignore": [
+            "**/*.*",
+            "**/*.html"
```

Comment:
This feels like a bad example - `**/*.html` is a subset of `**/*.*` and is therefore redundant.